concent_api/concent_api/tests/test_middleware.py

```diff
@@ -317,3 +318,13 @@ def test_that_empty_string_is_returned_when_accept_header_is_malformed(self):
         return_type = determine_return_type(request_headers)
 
         self.assertEqual(return_type, "")
+
+    def test_that_with_value_from_accept_header_values_transited_to_default_list_json_is_used(self):
+        for value in ACCEPT_HEADER_VALUES_TRANSITED_TO_DEFAULT:
+            request_headers = {
+                "HTTP_ACCEPT": value
```

Comment:
what would be desired behaviour with `HTTP_ACCEPT` like:
`"text/html, */*"`
`

Reply:
@dybi `best_match` returns `text/html` in that case.

Reply:
And that's correct. Spec says that the more specific types should be chosen (the one with less wildcards), which is `text/html` in this case.